src/deep-cache/lib/SharedCache.js

```diff
@@ -62,7 +63,7 @@ export class SharedCache {
     let action = request.action;
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: ['let' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).](https://www.codacy.com/app/MitocGroup/deep-framework/file/1784205094/issues/source?bid=2895475&fileBranchId=3068904#l63)